src/main/java/core/basesyntax/strategy/impl/SupplyService.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.Strategy;
+import java.util.Map;
+
+public class SupplyService implements Strategy {
+    @Override
+    public void processData(Map<String, Integer> fruitData, String fruit, int quantity) {
+        fruitData.put(fruit, fruitData.getOrDefault(fruit, 0) + quantity);
```

Comment:
nice way to work with the hashmap, use HashMap's `merge()` here and in return to simplify this even more